content/doc/book/managing/system-properties.adoc

```diff
@@ -47,542 +55,572 @@ If you find some of those useful, please file a ticket to promote it to the offi
-
-|jenkins.install.runSetupWizard |undefined |2.0 a|
-Set to `+false+` to skip install wizard. Note that this leaves Jenkins
-unsecured by default. +
-Development-mode only: Set to `+true+` to not skip showing the setup
-wizard during Jenkins development.
-
-NOTE: This property is only effective the first time you run Jenkins in
-given JENKINS_HOME.
-
-|jenkins.security.stapler.StaplerDispatchValidator.disabled |false
-|2.186 / 2.176.2 |`+true+`  to disable
-link:/security/advisory/2019-07-17/#SECURITY-534[the
-SECURITY-534 fix].
-
-|jenkins.security.ApiTokenProperty.adminCanGenerateNewTokens  |false
-|2.129 a|
-`+true+` to allow users with `+ADMINISTER+` permission to create API
-tokens using the new system for any user.
-
-NOTE: The user will not be able to use that token since it's only
-displayed to the creator, once.
-
-|jenkins.security.ApiTokenProperty.showTokenToAdmins |false |1.638 |True
-to show API tokens for users to administrators on the user configuration
-page. This was set to false as part of
-https://wiki.jenkins-ci.org/display/SECURITY/Jenkins+Security+Advisory+2015-11-11#JenkinsSecurityAdvisory2015-11-11-APItokensofotherusersavailabletoadmins[SECURITY-200]
-
-|jenkins.security.FrameOptionsPageDecorator.enabled |true |1.581
-|Whether to send `+X-Frame-Options: sameorigin+` header, set to false to
-disable and make Jenkins embeddable
-
-|jenkins.slaves.JnlpSlaveAgentProtocol3.enabled |undefined |1.653
-|`+false+` to disable the JNLP3 agent protocol, `+true+` to enable it.
-Otherwise it's randomly enabled/disabled to A/B test it.
-
-|jenkins.slaves.NioChannelSelector.disabled |false |1.560 |true to
-disable Nio for JNLP slaves
-
-|jenkins.ui.refresh |false |2.222 |`+true+` to enable the new experimental UX
-on Jenkins. See https://issues.jenkins-ci.org/browse/JENKINS-60920[JENKINS-60920].
-Also see https://www.jenkins.io/sigs/ux/[Jenkins UX SIG].
-
-|org.jenkinsci.main.modules.sshd.SSHD.idle-timeout |undefined |2.22
-|Allows to configure the SSHD client idle timeout (value in
-milliseconds). Default value is 10min (600000ms).
-
-|org.jenkinsci.plugins.workflow.steps.durable_task. +
-DurableTaskStep.REMOTE_TIMEOUT
-|20 seconds |workflow-durable-task-step-plugin 2.29 |How long to wait
-for remote calls (see
-https://issues.jenkins-ci.org/browse/JENKINS-46507[JENKINS-46507])
-|===
+* *$\{ITEM_ROOTDIR}* – The directory containing the job metadata within Jenkins home.
+* *$\{ITEM_FULL_NAME}* – The full name of the item, with file system unsafe characters replaced by others.
+* *$\{ITEM_FULLNAME}* – See above, but does not replace unsafe characters. This is a legacy option and should not be used.
+
++
+For instance, if you would like to store builds outside of Jenkins home, you can use a value like the following: `+/some_other_root/builds/${ITEM_FULL_NAME}+` This used to be a UI setting, but was removed in 2.119 as it did not support migration of existing build records and could lead to build-related errors until restart.
```

Comment:
Couldn't figure out a way that would separate this paragraph from the list above while retaining its inclusion in the definition list. Indenting this will break the continuation in the previous line.